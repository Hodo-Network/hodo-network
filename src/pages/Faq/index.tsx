import React from "react";

const headline = "Frequently Asked Questions";
const faqs = [
  {
    id: 1,
    question: "Which blockchain does the marketplace use?",
    answer:
      "The marketplace was built using the Avalanche blockchain. Assets are minted on the C-Chain.",
  },
  {
    id: 2,
    question:
      "Why were the assets minted on the C-Chain instead of the X-Chain?",
    answer:
      "The C-Chain is EVM compatible which will allow interoperability with other EVM chains in the future. Using the C-Chain also allows us to enable wallets like MetaMask for a better user experience.",
  },
];

export default function Faq() {
  return (
    <div className='p-4 sm:p-8 lg:p-12 max-w-8xl divide-y-2 divide-gray-200 dark:divide-gray-600'>
      <h2 className='text-3xl sm:text-4xl font-extrabold text-gray-900 dark:text-white'>
        {headline}
      </h2>
      <div className='mt-6'>
        <dl className='space-y-8 divide-y divide-gray-200 dark:divide-gray-700'>
          {faqs.map((faq) => (
            <div key={faq.id} className='pt-6 md:grid md:grid-cols-12 md:gap-8'>
              <dt className='text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5'>
                {faq.question}
              </dt>
              <dd className='mt-2 md:mt-0 md:col-span-7'>
                <p className='text-base text-gray-700 dark:text-gray-200'>
                  {faq.answer}
                </p>
              </dd>
            </div>
          ))}
        </dl>
      </div>
    </div>
  );
}
