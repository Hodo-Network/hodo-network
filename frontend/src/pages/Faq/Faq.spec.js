import ReactDOM from "react-dom";
import { cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import Faq from "./index";

afterEach(cleanup);

test("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Faq />, div);
});

test("matches snapshot", () => {
  const tree = renderer.create(<Faq />).toJSON();
  expect(tree).toMatchSnapshot();
});
