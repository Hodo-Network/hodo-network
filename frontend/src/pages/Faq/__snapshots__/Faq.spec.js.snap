// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="p-4 sm:p-8 max-w-8xl divide-y-2 divide-gray-200 dark:divide-gray-600"
>
  <h2
    className="text-3xl sm:text-4xl font-extrabold text-gray-900 dark:text-white"
  >
    Frequently Asked Questions
  </h2>
  <div
    className="mt-6"
  >
    <dl
      className="space-y-8 divide-y divide-gray-200 dark:divide-gray-700"
    >
      <div
        className="pt-6 md:grid md:grid-cols-12 md:gap-8"
      >
        <dt
          className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
        >
          Which blockchain does the marketplace use?
        </dt>
        <dd
          className="mt-2 md:mt-0 md:col-span-7"
        >
          <p
            className="text-base text-gray-700 dark:text-gray-200"
          >
            The marketplace was built using the Avalanche blockchain. Assets are minted on the C-Chain.
          </p>
        </dd>
      </div>
      <div
        className="pt-6 md:grid md:grid-cols-12 md:gap-8"
      >
        <dt
          className="text-base font-medium text-gray-800 dark:text-gray-100 md:col-span-5"
        >
          Why were the assets minted on the C-Chain instead of the X-Chain?
        </dt>
        <dd
          className="mt-2 md:mt-0 md:col-span-7"
        >
          <p
            className="text-base text-gray-700 dark:text-gray-200"
          >
            The C-Chain is EVM compatible which will allow interoperability with other EVM chains in the future. Using the C-Chain also allows us to enable wallets like MetaMask for a better user experience.
          </p>
        </dd>
      </div>
    </dl>
  </div>
</div>
`;
